apollo-router-core/src/json_ext.rs

```diff
@@ -520,7 +519,7 @@ impl fmt::Display for Path {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use serde_json::json;
+    use serde_json_bytes::bjson;
```

Comment:
:grey_question: wouldn't it be better to keep the name "json" for the macro and let the user alias the import if they need to? I imagine using serde_json and serde_json_bytes is less common than using one or the other exclusively?

Reply:
I think (not sure it's the case anymore) there are a few tests that used both, and it made more sense to have different names when I was testing from inside the router's code base. Now it could be renamed in serde_json_bytes, yes

Reply:
fixed in 4207563